sapp/ui/frontend/src/Filter.js

```diff
@@ -780,7 +784,57 @@ const SaveFilterModal = (
     return <Alert type="error">{error.toString()}</Alert>;
   }
 
-  const onOk = (): void => {
+  let filterList = [];
+
+  const importProps = {
+    multiple: true,
+    onRemove: filterFile => {
+      filterList.splice(filterList.indexOf(filterFile), 1);
+    },
+    beforeUpload: filterFile => {
+      if (filterFile.type !== 'application/json') {
+        message.error(`${filterFile.name} is not a json file`);
+      } else {
+        filterList = [...filterList, filterFile];
+      }
+      return filterFile.type === 'application/json' ?
+        false : Upload.LIST_IGNORE;
+    },
+    filterList,
+  };
+
+  const onImport = () => {
+    let count = filterList.length;
+    filterList.forEach(filterFile => {
+      let reader = new FileReader();
+      reader.onload = (e) => {
+        let filter: FilterDescription = JSON.parse(e.target.result);
+        if(filter["name"] !== undefined &&
```

Comment:
Formatting here looks off (space after `if`), this might be something we'd want to add to the open source workflow (I'm not sure what tool we're using internally, @0xedward maybe?).

Reply:
I'm not sure if our internal linter catches whitespace issues, but there's a [PR](https://github.com/facebook/sapp/pull/32) to add ESLint where we can see if we can include a rule to cover this though.

The ESLint config in the PR extends the [recommended rule set](https://eslint.org/docs/rules/) so I think it should cover this case for us